dags/atd_moped_ecapris_status_sync.py

```diff
@@ -0,0 +1,81 @@
+# Test locally with: docker compose run --rm airflow-cli dags test atd_moped_ecapris_status_sync
+
+import os
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration
+
+from utils.onepassword import get_env_vars_task
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "retry_delay": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+REQUIRED_SECRETS = {
+    "HASURA_ENDPOINT": {
+        "opitem": "Moped Hasura Admin",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Endpoint",
+    },
+    "HASURA_ADMIN_SECRET": {
+        "opitem": "Moped Hasura Admin",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Admin Secret",
+    },
+    "ORACLE_USER":{
+        "opitem": "Finance Data Warehouse Oracle DB",
+        "opfield": "production.Username",},
+    "ORACLE_PASSWORD": {
+        "opitem": "Finance Data Warehouse Oracle DB",
+        "opfield": "production.Password",
+    },
+    "ORACLE_HOST": {
+        "opitem": "Finance Data Warehouse Oracle DB",
+        "opfield": "production.Host",
+    },
+    "ORACLE_PORT": {
+        "opitem": "Finance Data Warehouse Oracle DB",
+        "opfield": "production.Port",
+    },
+    "ORACLE_SERVICE": {
+        "opitem": "Finance Data Warehouse Oracle DB",
+        "opfield": "production.Service",
+    }, 
+}
+
+
+with DAG(
+    dag_id="atd_moped_ecapris_status_sync",
+    description="sync eCapris statuses to Moped database",
+    default_args=DEFAULT_ARGS,
+    schedule_interval="*/30 * * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
+    dagrun_timeout=duration(minutes=30),
+    tags=["repo:atd-moped", "moped", "ecapris"],
+    catchup=False,
+) as dag:
+    docker_image = f"atddocker/atd-moped-etl-ecapris-statuses:{DEPLOYMENT_ENVIRONMENT}"
+
+    env_vars = get_env_vars_task(REQUIRED_SECRETS)
+
+    t1 = DockerOperator(
+        task_id="ecapris_statues_to_moped",
```

Comment:
a lil typo!

Reply:
hahaha eCapris statues 🗽 thanks! just fixed this 🙏